README.md

```diff
@@ -1,130 +1,144 @@
-- Ingest the iris data.
+- Create an Iris data-set generator (ingestion) function.
+- Ingest the Iris data set.
 - Analyze the data-set features.
 - Train and test the model using multiple algorithms (AutoML).
 - Deploy the model as a real-time serverless function.
-- Test the serverless function REST API with a test data set.
+- Test the serverless function's REST API with a test data set.
 
-To start, download the notebook [sklearn-project.ipynb](./sklearn-pipe/sklearn-project.ipynb) into an empty directory and run the cells one by one.
+To run the demo, download the [**sklearn-project.ipynb**](./sklearn-pipe/sklearn-project.ipynb) notebook into an empty directory and execute the cells sequentially.
 
 <br><p align="center"><img src="./docs/trees.png" width="500"/></p><br>
 
 <a id="demo-sklearn-pipe-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
 
-You can see various output plots in [sklearn-pipe/plots](sklearn-pipe/plots) (static HTML files).
+The output plots can be viewed as static HTML files in the [sklearn-pipe/plots](sklearn-pipe/plots) directory.
 
 <br><p align="center"><img src="./docs/skpipe.png" width="500"/></p><br>
 
 <a id="demo-horovd-image-classification"></a>
-## [Image Classification Using Distributed Training (Horovod)](horovod-pipe/horovod-project.ipynb)
+## Horovod Image-Classification Demo &mdash; Distributed Training
+
+The [**horovod-pipe**](horovod-pipe/README.md) demo demonstrates an end-to-end image-classification solution using [TensorFlow](https://www.tensorflow.org/) (versions 1 or 2), [Keras](https://keras.io/), [Horovod](https://eng.uber.com/horovod/), and [Nuclio](https://nuclio.io/).
 
-This example uses TensorFlow (v1 or v2), Horovod, and Nuclio, demonstrating end-to-end solution for image classification.
-The demo consists of four MLRun and Nuclio functions and Kubeflow Pipelines orchestration:
+The demo consists of four MLRun and Nuclio functions and a Kubeflow Pipelines orchestration:
 
-1. **Download**: Import an image archive from S3 to the cluster file system.
+1. **Download**: Import an image archive from AWS S3 to your cluster's data store.
 2. **Label**: Tag the images based on their name structure.
-3. **Training**: Distributed training using TF1 or TF2, Keras and Horovod.
-4. **Inference**: Automated deployment of Nuclio model serving function.
+3. **Training**: Perform distributed training using TensorFlow, Keras, and Horovod.
+4. **Inference**: Automate deployment of a Nuclio model-serving function.
 
-> **Note:** The demo supports both TensorFlow v1 and v3, there is one (shared) notebook and two code files (one per TF version)
+> **Note:** The demo supports both TensorFlow versions 1 and 2.
+> There's one shared notebook and two code files &mdash; one for each TensorFlow version.
 
 <br><p align="center"><img src="./docs/hvd-flow.png" width="600"/></p><br>
 
 <a id="demo-horovd-image-classification-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
```

Comment:
@Sharon-iguazio same as above

Reply:
Changed to bold — see [my response](https://github.com/mlrun/demos/pull/40#discussion_r440025898) for your similar previous comment.